### ISSUE TITLE
Avoid device/application dependent event handlers

### ISSUE THREAD

**Issue by alvaromontoro** [jackdomleo7/Checka11y.css#48]
### Describe the new a11y feature or project enhancement

All the page functionality must be available via keyboard, but if elements have mouse events associated, those event handlers may not be triggered correctly with the keyboard. For example:

```html
<div onclick="doSomething()">This div is clickable but the effect cannot be triggered via keyboard</div>
```

Apart from not being accessible via keyboard, many times these issues are a reflection of a different underlying problem (i.e., the `<div>` above should probably be a button.) 

### Describe the solution you'd like

Display a warning if there's an element that has mouse event handlers associated with it. The ones that are added via JavaScript are not reachable via CSS, but if the event is inline, it can be targeted with CSS. E.g., the following rule will apply styles to the `<div>` above:

```css
[onclick],
[ondblclick],
[onmouseover],
[onmouseenter],
[onmouseleave],
[onmousedown],
[onmouseup] {
  /* warning/error */
}
```

And it could be extended to avoid false positives. For example, by discarding elements that would actually trigger the mouse event via keyboard like a button or a link (e.g.: `[onclick]:not(a):not(button)`)

### Link(s)

- [Success criterion 2.1.1: Keyboard](https://www.w3.org/TR/WCAG21/#keyboard)

**Comment by jackdomleo7** [comment]
Ah this seems really good! What error/warning message would you show to the user/developer? 🙂

**Comment by alvaromontoro** [comment]
The message that WebAIM shows is: "An event handler is present that may not be accessible. [...] The JavaScript events in use do not appear to be accessible to both mouse and keyboard users. To be fully accessible, critical JavaScript interaction should be device independent."

**Comment by alvaromontoro** [comment]
#59 is a similar issue and it has a long discussion with different points not covered here.
